<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGui</class>
 <widget class="QWidget" name="TestGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
   <item>
    <widget class="QTextBrowser" name="out"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Modem</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbModemPoweredOn">
          <property name="text">
           <string>PoweredOn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbModemPoweredOff">
          <property name="text">
           <string>PoweredOff</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbModemOnline">
          <property name="text">
           <string>Online</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbModemOffline">
          <property name="text">
           <string>Offline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbModemLockdown">
          <property name="text">
           <string>Lockdown</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbModemLockup">
          <property name="text">
           <string>Lockup</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>NetworkRegistration</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pbNetworkUnregistered">
          <property name="text">
           <string>Unregistered</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbNetworkRegistered">
          <property name="text">
           <string>Registered</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbNetworkScan">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>ConnectionContext</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pbContextAPN">
          <property name="text">
           <string>APN</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="contextAPN"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbContextUsername">
          <property name="text">
           <string>User</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="contextUsername"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pbContextPassword">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="contextPassword"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pbContextActive">
          <property name="text">
           <string>Active</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
